web/src/pages/orders/[orderId]/index.page.jsx

```diff
@@ -16,63 +29,47 @@ const Order = () => {
       href: '#',
       title: 'Order Detail',
     },
-  ]
-
-  const orderDetails = {
-    date: 'January 6, 2024',
-    jobOrder: '819089',
-    status: 'In-Transit',
-    category: 'Battery',
-    brand: 'ASUS',
-    partModel: 'B31N1726',
-    linkRef: 'https://www.amazon.com/B31N1726...',
-    downPayment: '500',
-    branch: 'Davao',
-    unitModel: 'Rog Strix',
-    comment: 'Need by the customer before 4th week of Jan. 2024',
-  }
-  const imageUrls = [
-    'https://m.media-amazon.com/images/W/MEDIAX_792452-T2/images/I/71k8dZdfStS._AC_SL1500_.jpg',
-    'https://m.media-amazon.com/images/W/MEDIAX_792452-T2/images/I/61CWkh-2YWS._AC_SL1500_.jpg',
-    'https://m.media-amazon.com/images/W/MEDIAX_792452-T2/images/I/61uR43INXvS._AC_SL1500_.jpg',
-    'https://m.media-amazon.com/images/W/MEDIAX_792452-T2/images/I/81AESsFpWFS._AC_SL1500_.jpg',
-  ]
+  ];
 
+  
   return (
     <Template>
       <PageHeader breadcrumbs={breadcrumbs} />
       <section className='bg-white dark:bg-gray-900'>
```

Comment:
Place isLoading here, I already created a component for that
```suggestion
      <PageHeader breadcrumbs={breadcrumbs} />
      {
      isloading ? 
      <Loading />
      :
      (
        <section className='bg-white dark:bg-gray-900'> 
        ...
        </section
      )
      }
      
```